### ISSUE TITLE
Loss documentation is wrong? Loss function actually returns means over a batch

### ISSUE THREAD

**Issue by worthy7** [keras-team/keras#20395]
https://keras.io/api/losses/#standalone-usage-of-losses

It states: `By default, loss functions return one scalar loss value per input sample, e.g.`

But the example is passing 4 samples `ops.ones((2, 2,))` , and returning 2 values `<Array: shape=(2,), dtype=float32, numpy=array([1., 1.], dtype=float32)>`

So which is it?

**Comment by jeffcarp** [comment]
Thanks for pointing this out. We can make the docs clearer. If you want different behavior, constructing the loss with `reduction=None` will do what you originally expected.
